README.md

```diff
@@ -63,6 +63,30 @@ restart the server. See the [rack_mini_profiler] gem for more details.
 [Laptop]: https://github.com/18F/laptop
 [rack_mini_profiler]: https://github.com/MiniProfiler/rack-mini-profiler
 
+#### Testing Analytics
+
+If you want to visualize and query the event and log data, you can install
+Elasticsearch, Logstash, and Kibana. On OS X, the easiest way is with Homebrew:
+
+```
+brew tap homebrew/services
+
+brew install elasticsearch logstash kibana
+
+brew services start elasticsearch
+brew services start kibana
+
+# Start logstash by running this command from this repo's root directory:
+logstash -f logstash.conf
+```
+
+Change the `logstash_type` entry in `config/application.yml` to `'udp`'
+and restart the Rails server (ctrl-c, then `foreman start`).
```

Comment:
These instructions work fine for me. Of note is that you need to make sure your elasticsearch version is up to date since ELK versions must be all in sync. I had a pre-existing elasticsearch install that required updating.